CHANGELOG.md

```diff
@@ -0,0 +1,44 @@
+# Changelog
+
+All notable changes to this project will be documented in this file.
+
+The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.1.0/),
+and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
+
+## [Unreleased]
+
+### Added
```

Comment:
Where possible, it'd be nice to add a link to the implementing PR for each of these, as it tends to be super helpful to those reading who want more details about a particular change.

Reply:
Those are useful indeed and helped me many times, for example when the minimal versions job fails one will need to go and find the lib version where the change in question has been introduced - it just saves a lot of time.